eng/common/scripts/Verify-Links.ps1

```diff
@@ -286,9 +299,12 @@ while ($pageUrisToCheck.Count -ne 0)
   Write-Host "Found $($linkUris.Count) links on page $pageUri";
   $badLinksPerPage = @();
   foreach ($linkUri in $linkUris) {
-    $linkUri = ReplaceGithubLink $linkUri
-    $isLinkValid = CheckLink $linkUri
+    $replacedLink = ReplaceGithubLink $linkUri
```

Comment:
I assume this is to fix the logging issue where we log with the replaced link when it isn't found? I'm fine with this change but just wanted to confirm this is the reason.

Reply:
Right, I associate the issue in description. It is a small fix. Don't want to start a new thread. :p